src/NServiceBus.AcceptanceTests/EndpointTemplates/EndpointCustomizationConfigurationExtensions.cs

```diff
@@ -15,7 +15,9 @@ public static IEnumerable<Type> GetTypesScopedByTestClass(this EndpointCustomiza
 
             var assembliesToScan = assemblies.Assemblies
                 //exclude acceptance tests by default
-                .Where(a => a != Assembly.GetExecutingAssembly()).ToList();
+                .Where(a => a != Assembly.GetExecutingAssembly())
+                .Where(a => !a.GetName().Name.EndsWith("Tests"))
```

Comment:
Is this a change that we want to keep, or should this also be reverted? Seems like part of your debugging changes.

Reply:
Wasn't sure whether to keep this one or not. Technically this would still be correct as the assembly scanning would look scan messages in every assembly found but so far there should only be one "Tests" assembly per test project. Referencing another test project seems to also confuse the build and wouldn't really be supported so we could remove this again. Thoughts?

Reply:
I'm leaning towards removing it.